symplyphysics/laws/hydro/input_force_to_area_ratio_equals_to_output.py

```diff
@@ -14,6 +14,13 @@
 ## S1 is the area of the first piston
 ## S2 is the area of the second piston
 
+## Proof
+## The pressure that piston number one creates is equal to: p1 = F1 / S1
+## The pressure of the second piston on the liquid is: p2 = F2 / S2
+## If the pistons are in equilibrium, then the pressures p1 and p2 are equal, therefore, we can equate the right-hand sides of the expressions
+## ==> F1 / S1 = F2 / S2
```

Comment:
Delete. As I told, I want to see it as code, not comments in the code. You can see an example of such proof here: https://github.com/blackyblack/symplyphysics/blob/a3df72abb95b37aada38e6929dfc1153b032f873/symplyphysics/laws/gravity/free_fall_acceleration_from_height.py#L27